scripts/sct_get_centerline.py

```diff
@@ -125,11 +125,18 @@ def run_main():
     if method == 'viewer':
         # Manual labeling of cord centerline
         im_labels = _call_viewer_centerline(Image(fname_data), interslice_gap=interslice_gap)
-    else:
+    elif method == 'fitseg':
+        im_labels = Image(fname_data)
+    elif method == 'optic':
         # Automatic detection of cord centerline
         im_labels = Image(fname_data)
         param_centerline.algo_fitting = 'optic'
         param_centerline.contrast = contrast_type
+    else:
+        error = 'ERROR: please refer to help'
+        sct.printv(error, type='error')
```

Comment:
could save one line of code-- and make message more explicit
```suggestion
        sct.printv("ERROR: The selected method is not available: {}".format(method), type='error')
```